verilog/CST/macro.cc

```diff
@@ -83,10 +83,10 @@ const verible::SyntaxTreeLeaf& GetMacroArgName(
   return GetSubtreeAsLeaf(macro_formal_arg, NodeEnum::kMacroFormalArg, 0);
 }
 
-const verible::SyntaxTreeLeaf& GetFileFromPreprocessorInclude(
+const verible::Symbol& GetFileFromPreprocessorInclude(
     const verible::Symbol& preprocessor_include) {
-  return GetSubtreeAsLeaf(preprocessor_include, NodeEnum::kPreprocessorInclude,
-                          1);
+  return *ABSL_DIE_IF_NULL(verible::GetSubtreeAsSymbol(
```

Comment:
Is it ok to add a check here?

Reply:
yes. after all the include must contain some information after `include` keyword.

Reply:
Yes, and ideally, this should be `CHECK`ed on CST node construction for includes.  Checking here will do for now.